### PR TITLE
override_values and prettier diffs for foreignkeys

### PR DESCRIPTION
**Pull request by jleclanche**
Apologies in advance for not splitting the PR.

The first commit implements override_values, a setting which lets a user override all values for a specific field at import time by specifying said value in the import form. This supports foreign keys with a similar UI as the django admin has.

The second commit improves diff of foreign keys. I tried to implement this in ForeignKeyWidget.export() but I couldn't figure out how, it's very hard to follow the code past a certain point.

Feedback welcome

### CODE REVIEW

**Comment by jleclanche** [comment]
I don't understand why travis is failing...

**Comment by teshomem** [comment]
@jleclanche I would appreciate if you could update me your plan on this.

**Comment by jleclanche** [comment]
@teshomem With no news from the repository owner, there isn't much I can do to get it merged unfortunately.

**Comment by bmihelac** [comment]
Unfortunately there are multiple failing tests and I do not have a time to devote at the moment.

**Comment by teshomem** [comment]
@bmihelac I need your help on this. Do you need any support for this? This is very important for me this time and please let me know your decision. Thanks.
